qml/components-v-play/Theme.qml

```diff
@@ -19,6 +19,8 @@ QtObject {
             return IconType.list
         case "about":
             return IconType.questioncircle
+        case "mapmarker":
+            return IconType.mapmarker
```

Comment:
We already return `"IconType.mapmarker"` under `case: "location"`. The naming for the cases is from Sailfish, because Sailfish first!